crawl-ref/source/evoke.cc

```diff
@@ -415,72 +415,10 @@ void wind_blast(actor* agent, int pow, coord_def target)
 
     for (actor *act : act_list)
     {
-        wind_beam.target = act->pos();
-        wind_beam.fire();
-
-        int push = _gale_push_dist(agent, act, pow);
-        bool pushed = false;
-
-        for (unsigned int j = 0; j < wind_beam.path_taken.size() - 1 && push;
-             ++j)
-        {
-            if (wind_beam.path_taken[j] == act->pos())
-            {
-                coord_def newpos = wind_beam.path_taken[j+1];
-                if (!actor_at(newpos) && !cell_is_solid(newpos)
-                    && act->can_pass_through(newpos)
-                    && act->is_habitable(newpos))
-                {
-                    act->move_to_pos(newpos);
-                    if (act->is_player())
-                        stop_delay(true);
-                    --push;
-                    pushed = true;
-                }
-                else //Try to find an alternate route to push
-                {
-                    bool success = false;
-                    for (adjacent_iterator di(newpos); di; ++di)
-                    {
-                        if (adjacent(*di, act->pos())
-                            && di->distance_from(agent->pos())
-                                == newpos.distance_from(agent->pos())
-                            && !actor_at(*di) && !cell_is_solid(*di)
-                            && act->can_pass_through(*di)
-                            && act->is_habitable(*di))
-                        {
-                            act->move_to_pos(*di);
-                            if (act->is_player())
-                                stop_delay(true);
-
-                            --push;
-                            pushed = true;
-
-                            // Adjust wind path for moved monster
-                            wind_beam.target = *di;
-                            wind_beam.fire();
-                            success = true;
-                            break;
-                        }
-                    }
-
-                    // If no luck, they slam into something.
-                    if (!success)
-                        collisions.insert(make_pair(act, newpos));
-                }
-            }
-        }
-
-        if (pushed)
+        if (agent->can_see(*act))
         {
-            if (act->is_monster())
-            {
-                act->as_monster()->speed_increment -= random2(6) + 4;
-                if (you.can_see(*act))
-                    affected_monsters.push_back(act->as_monster());
```

Comment:
This got lost, meaning that fan of gales will never print anything about monsters being blown back. (`player_affected` is never set, either.)